api/edge_api/management/commands/migrate_to_edge_v2.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+from django.core.management import BaseCommand
+
+from projects.models import EdgeV2MigrationStatus, Project
+from projects.tasks import migrate_project_environments_to_v2
+
+
+class Command(BaseCommand):
```

Comment:
Should this be an admin view rather than a command? How are we planning to trigger this?

Reply:
Create an ECS task, either manually or by adding it to deployment definition. It's a one-off task so either way is fine by me.